doc/sphinx/language/gallina-extensions.rst

```diff
@@ -296,8 +291,8 @@ the folded version delta-reduces to the unfolded version. This allows to
 precisely mimic the usual unfolding rules of constants. Projections
 obey the usual ``simpl`` flags of the ``Arguments`` command in particular.
 There is currently no way to input unfolded primitive projections at the
-user-level, and one must use the :flag:`Printing Primitive Projection Compatibility`
-to display unfolded primitive projections as matches and distinguish them from folded ones.
+user-level, and there is no way to display unfolded projections differently
+from folded ones.
 
```

Comment:
I just noticed this change as sphinx was failing due to missing ref. This is pretty bad IMO, maybe we should delay the PR.

Reply:
Impressive what the user documentation (with fatal warnings) can do to help the developers :smiley: